apps-rendering/src/components/editions/galleryImage/index.tsx

```diff
@@ -50,6 +58,23 @@ const styles = css`
 	}
 `;
 
+const imgStyles = css`
+	${from.tablet} {
+		flex: 0 0 538px;
```

Comment:
probably nothing to change here, but I'd like to understand a bit more about which cases we use `px` vs `rem` units